Makefile

```
@@ -18,10 +18,8 @@ SHELL := /usr/bin/env bash
 # Docker build config variables
 CREDENTIALS_PATH ?= /cft/workdir/credentials.json
 DOCKER_ORG := gcr.io/cloud-foundation-cicd
-DOCKER_TAG_BASE_KITCHEN_TERRAFORM ?= 2.0.0
+DOCKER_TAG_BASE_KITCHEN_TERRAFORM ?= 2.0.1
```

As per https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/217:

```suggestion
DOCKER_TAG_BASE_KITCHEN_TERRAFORM ?= 2.1.0
```